xdg-terminal-exec

```diff
@@ -290,11 +278,86 @@ read_entry_path() {
 	return 0
 }
 
-# See read_entry_path and check_entry
-if find_preferred_entry > /dev/null || find_any_entry > /dev/null; then
-	# Default to '-e' if unset
-	EXECARG=${EXECARG-'-e'}
-fi
+# Subfolder within $XDG_DATA_*, analoguous to 'applications/' used with desktop entry files
+DATA_PREFIX_DIR=xdg-terminals
+# Name of config files to search and read
+CONFIG_NAME=xdg-terminals.list
+
+# Directory hierarchy searched for terminal config and desktop entry files
+# TODO: Consider using $XDG_DATA_DIRS
```

Comment:
What do you mean here? There is already `$XDG_DATA_DIRS`  in the list.

Reply:
I can't remember, I've probably misread something
I'll remove the TODO